blog-cse/2023-03-31-content.md

```diff
@@ -0,0 +1,32 @@
+---
+title: March 31, 2023 - Content Release
+hide_table_of_contents: true
+image: https://help.sumologic.com/img/sumo-square.png
+authors:
+  - url: https://help.sumologic.com/release-notes-cse/rss.xml
+    image_url: /img/release-notes/rss-orange.png
+---
+
+Within this content release we have deprecated two of our First Seen rules linked to low fidelity as we continue to perform internal testing around similar detections. Additionally, we are bringing out a new set of Carbon Black mappers, expanding on our existing normalization with the product.
+
+#### Rules
+[Deleted] FIRST-S00011 First Seen Sysmon IMPHASH - Global
+[Deleted] FIRST-S00012 First Seen Sysmon IMPHASH - Host
```

Comment:
@ptobia , I think these (and the other sections) are supposed to be bulleted

Reply:
Agreed.

Reply:
will fix.